### ISSUE TITLE
How to change colima config default image

### ISSUE THREAD

**Issue by tibbus** [abiosoft/colima#995]
### Description

I just installed colima on macOS and I have an issue that it cannot download the image when I do "colima start":

```
    failed to download "https://github.com/abiosoft/colima-core/releases/download/ve.6.8-2/ubuntu-23.1
O-minimal-cloudimg-arm64.qcow2": Get "https://github.com/abiosoft/colima-core/releases/download/v0.6
```
This is happening because this is a work laptop and due to security reasons, I am not allowed to download it via the Terminal. I can download this file in the browser.

How can I change the colima config to use a local image instead of downloading this one?

I've tried to add something like this to .colima/default/colima.yaml but still wants to download it:

```
    images:
- location: *./ubuntu-23.10-minimal-cloudimg-amd64. raw°
arch: x86_64°
vmType: 'vz'
mountType: 'virtiofs'
```
Any ideas?

### Version

colima version HEAD-9b0809d
git commit: 9b0809d0ed9ad3ff1e57c405f27324e6298ca04f
limactl version 0.20.1
gemu-img version 8.2.1
Copyright (c) 2003-2023 Fabrice Bellard and the QEMU Project developers

### Operating System

- [ ] macOS Intel <= 13 (Ventura)
- [ ] macOS Intel >= 14 (Sonoma)
- [X] Apple Silicon <= 13 (Ventura)
- [ ] Apple Silicon >= 14 (Sonoma)
- [ ] Linux

### Output of `colima status`

colima is not running

### Reproduction Steps

1.
2.
3.

**Comment by roman-kiselenko** [comment]
It looks like a bug in `colima`, the `lima` handle `location` property and allow user to specify the `img` location.

```sh
cat ~/.colima/_lima/colima-docker/lima.yaml | grep location
    - location: ~/Downloads/data
➜  ~ colima start docker --edit
INFO[0000] editing in nvim from $EDITOR environment variable
INFO[0002] starting colima [profile=docker]
INFO[0002] runtime: docker
INFO[0003] starting ...                                  context=vm
> Using the existing instance "colima-docker"
> Starting the instance "colima-docker" with VM driver "qemu"
> QEMU binary "/opt/homebrew/bin/qemu-system-aarch64" seems properly signed with the "com.apple.security.hypervisor" entitlement
> "Attempting to download the image" arch=aarch64 digest="sha512:00e3339bdebd98c3e003570ffb3ad4b01630fe4fcecd15061d5d58e14c07b211c718ed20fa3e4cce227d3b1c59fd98241eaa3e9e2cdfa04acfe32b4bc385428c" location="https://github.com/abiosoft/colima-core/releases/download/v0.6.8-2/ubuntu-23.10-minimal-cloudimg-arm64.qcow2"
> Downloading the image (ubuntu-23.10-minimal-cloudimg-arm64.qcow2)
```
xref https://github.com/lima-vm/lima/issues/2226

**Comment by roman-kiselenko** [comment]
Related #985 🤔

**Comment by abiosoft** [comment]
> It looks like a bug in colima, the lima handle location property and allow user to specify the img location.

It is not a bug, it is the expected behaviour. 

The Colima disk image includes bundled dependencies which may be missing in custom images.

However, providing an option for users to override the image download URL is being considered.

**Comment by tibbus** [comment]
sure

multumesc,
Tiberiu Popescu

**Comment by abiosoft** [comment]
Ability to specify a locally downloaded disk image has been implemented in #1216

**Comment by PylotLight** [comment]
I assume this means we can alter the make file to use debian instead, although ideally it would be nice to have a flag for this instead of debian if we get around to it?
Would we be interested in a PR on this?
Looking at the available images, would the next closest to ubuntu minimal be genericcloud image?
https://cloud.debian.org/images/cloud/bookworm/latest/

Anything else worth noting while I try get a debian image built for this instead we can use?
